src/views/ExternalAccess/ExternalAccess.module.css

```diff
@@ -31,6 +31,13 @@
   cursor: pointer;
 }
 
+.host {
+  word-wrap: break-word;
+  overflow: hidden;
```

Comment:
With the break-word above, do we need hidden?

Reply:
I guess it is needed to tell the browser the importance of wrapping this content… Let’s keep it!